winit/src/settings.rs

```diff
@@ -158,12 +173,16 @@ impl Window {
         ))]
         {
             // `with_name` is available on both `WindowBuilderExtWayland` and `WindowBuilderExtX11` and they do
-            // exactly the same thing. We arbitrarily choose `WindowBuilderExtWayland` here.
-            use ::winit::platform::wayland::WindowBuilderExtWayland;
+            // exactly the same thing. We arbitrarily choose `WindowBuilderExtX11` here.
+            use winit::platform::x11::WindowBuilderExtX11;
```

Comment:
iced_winit defines features `x11` and `wayland` but it assumes that they are always set. Afaik they will always be set if iced_winit is used as an indirect dependency of iced. 
But is that intended or should users of iced be able to disable one of these features? If so, we'd also need to put the use here behind a #[cfg()]